ee/backend/controllers/bitbucket.go

```
@@ -0,0 +1,340 @@
+	}
+
+	verifySignature(c, bodyBytes, bitbucketWebhookSecret)
+
+	switch eventKey {
+	case "pullrequest:comment_created":
+		err := json.Unmarshal(bodyBytes, &pullRequestCommentCreated)
+		if err != nil {
+			log.Printf("error parsing pullrequest:comment_created event: %v", err)
+			log.Printf("error parsing pullrequest:comment_created event: %v", err)
+		}
+		go handleIssueCommentEventBB(ee.BitbucketProvider, &pullRequestCommentCreated, ee.CiBackendProvider, orgId, bitbucketAccessToken)
```

golangci-lint (1.62.2)

Error return value is not checked

(errcheck)